### PR TITLE
Make the codegen more succinct

### PR DESCRIPTION
**Pull request by snawaz**
I think it cannot be made any more smaller.. now!

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -183,7 +183,7 @@ fn generate_cargo_nix(mut out: impl io::Write) {
                 writeln!(f, "let")?;
                 writeln!(
                     f.indent(2),
-                    "drvs = genDrvsByProfile profilesByName (attrs: mkRustCrate (f attrs));"
+                    "drvs = genDrvsByProfile profilesByName ({{profile, profileName}}: mkRustCrate ({{ inherit {} profile; }} // (f profileName)));", scope.release
```
Comment (trha) [inline src/main.rs]: Style: 
```suggestion
                    "drvs = genDrvsByProfile profilesByName ({{ profile, profileName }}: mkRustCrate ({{ inherit {} profile; }} // (f profileName)));", scope.release
```

**Comment by snawaz** [comment]
@trha : please dont be confused by `features` formatting with the correct spacing now. it was done in the previous PR but I didn't regenerate it. So now they have all the changes.

File: src/main.rs (hunk 1 above)
Comment (snawaz) [reply src/main.rs]: Done. Rebased on master. And pushed.

**Comment by trha** [comment]
Oops, I didn't mean to approve. Please update the generated file also.

**Pull request merged**